examples/cr.yaml

```diff
@@ -16,8 +16,5 @@ spec:
   managementState: Managed
   # there should be only one console at this time
   count: 1
-  # this is the console image
-  baseImage: docker.io/openshift/origin-console
```

Comment:
Eliminated this, moved to using an `env var` on the operator deployment (similar to other operators)